hoot-core/src/main/cpp/hoot/core/io/OgrWriter.cpp

```diff
@@ -573,7 +573,7 @@ void OgrWriter::_writePartial(ElementProviderPtr& provider, const ConstElementPt
 
     try
     {
-      g = ElementConverter(provider).convertToGeometry(e);
+      g = ElementConverter(provider).convertToGeometry(e, false);
```

Comment:
Adding `false` tells `convertToGeometry` to not throw an exception but create an empty geometry.  In the case of writing an OGR file, I think that is the right thing to do.